### PR TITLE
FUSETOOLS2-448 - environment variables should be only defined as key: value type

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: src/commands/ConvertEnvironmentVariableCommand.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+		const success: boolean = await createEnvironmentVariable(envVarNode, refName, refType, dvConfig, envVarNode.getProject().getFile());
+		if (success) {
+			vscode.window.showInformationMessage(`Datasource ${envVarNode.key} has been migrated to ${refType}...`);
+		} else {
+			vscode.window.showErrorMessage(`An error occured when trying to migrate datasource with name ${envVarNode.key} to a reference file of type ${refType}...`);
+		}
+	}
+}
+
+export async function createEnvironmentVariable(envVarNode: EnvironmentVariableTreeNode, refName: string, refType: string, dvConfig: DataVirtConfig, file: string): Promise<boolean> {
```
Comment (bfitzpat) [inline src/commands/ConvertEnvironmentVariableCommand.ts:55]: Should this be convert and not create?

Comment (lhein) [reply src/commands/ConvertEnvironmentVariableCommand.ts]: fixed

**Comment by bfitzpat** [comment]
Looks like an issue with one of your regexes -- 

```
Failed to parse file: /home/travis/build/jboss-fuse/vscode-datavirt/out/utils.js

Transformation error; return original code

Error: Line 89: Invalid regular expression
```

**Comment by bfitzpat** [comment]
Extra validation works great now!

**Pull request merged**